ONSdigital/ras-rm-auth-service
# Due Deletion scheduler for account not accessed for a months.

# Motivation and Context
* create a scheduler to provide a due deletion notification to the users when their account has not be accessed for a duration.

# What has changed
* notification scheduler job has been provided to carry out the tasks. 
* code refactor has been done.
# How to test?
* we will have to create a test suite data to make sure all the scheduler provided carries out the tasks without any issues.
